config/default.yml

```diff
@@ -99,6 +99,11 @@ Minitest/AssertWithExpectedArgument:
   Safe: false
   VersionAdded: '0.11'
 
+Minitest/EmptyLinesBeforeAssertionMethod:
```

Comment:
Yeah, I was wondering about the name of this cop. I've renamed it to `Minitest/EmptyLineBeforeAssertionMethods`.

Reply:
Thinking about this again, I wonder if talking about line/lines is too specific, and the a name should better convey the intention.

This is closely tied to [Arrange-Act-Assert](https://automationpanda.com/2020/07/07/arrange-act-assert-a-pattern-for-writing-good-tests/) or the [Four Phase Test](https://thoughtbot.com/blog/four-phase-test).

I can't think of a good name yet, but maybe something along the lines of `SeparateAssertionPhase` or `DistinctAssertions`...

Reply:
@andyw8 I was second thoughts about this cop name. But I will name it as `Minitest/EmptyLineBeforeAssertionMethods` because the RuboCop core has many similar names.
The attractive candidate `SeparateAssertionPhase` has been included as the cop document description.